src/main/kotlin/app/revanced/patches/instagram/patches/ads/timeline/patch/HideTimelineAdsPatch.kt

```diff
@@ -61,15 +61,18 @@ class HideTimelineAdsPatch : BytecodePatch(
             val returnFalseLabel = "an_ad"
 
             val checkForAdInstructions =
-                listOf(GenericMediaAdFingerprint, PaidPartnershipAdFingerprint, ShoppingAdFingerprint)
+                listOf(GenericMediaAdFingerprint, ShoppingAdFingerprint)
                     .map(MediaAdFingerprint::toString)
                     .joinToString("\n") {
                         """ 
                             invoke-virtual {v$mediaInstanceRegister}, $it
                             move-result v$freeRegister
                             if-nez v$freeRegister, :$returnFalseLabel
                         """.trimIndent()
-                    }.let { "$it\nconst/4 v0, 0x1\nreturn v0" }
+                    }.let { "$it\n" +
+                            "invoke-static {v$mediaInstanceRegister}, ${PaidPartnershipAdFingerprint.toString()}\n" +
+                            "move-result v$freeRegister\n" +
+                            "if-nez v$freeRegister, :$returnFalseLabel\nconst/4 v0, 0x1\nreturn v0" }
```

Comment:
Can you document the new patching strategy as comments? I am having a hard time following the new strategy